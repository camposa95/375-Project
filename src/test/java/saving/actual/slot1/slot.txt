gameType: Beginner
numPlayers: 2
language: English
