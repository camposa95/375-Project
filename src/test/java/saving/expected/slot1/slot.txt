gameType: Beginner
numPlayers: 4
language: English
